### PR TITLE
Multistage onboarding set default v1 experiment recipe

### PR DESCRIPTION
**Pull request by punamdahiya**

### CODE REVIEW

File: messaging-experiments-82.json [hunk 1]
```diff
@@ -1058,5 +1058,888 @@
         }
       }
     ]
+  },
+  {
+    "id": "MULTI-STAGE-ABOUTWELCOME-SET-DEFAULT-V1",
+    "enabled": true,
+    "targeting": "firefoxVersion >= '82.' && browserSettings.update.channel != 'release' && region == 'US' && localeLanguageCode == \"en\" && ((isFirstStartup && !('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)) || 'bug-1675623-message-multi-stage-onboarding-set-default-experiment-release-82-85' in activeExperiments) && 'app.shield.optoutstudies.enabled'|preferenceValue",
+    "arguments": {
+      "slug": "bug-1675623-message-multi-stage-onboarding-set-default-experiment-release-82-85",
```
Comment (ncloudioj) [inline messaging-experiments-82.json:1067]: 79 characters 👍

**Review by ncloudioj — APPROVED** [review]
LGTM, thanks!

This single recipe gets 900 lines now 😅

File: messaging-experiments-82.json [hunk 2]
```diff
@@ -1058,5 +1058,888 @@
         }
       }
     ]
+  },
+  {
+    "id": "MULTI-STAGE-ABOUTWELCOME-SET-DEFAULT-V1",
+    "enabled": true,
+    "targeting": "firefoxVersion >= '82.' && browserSettings.update.channel != 'release' && region == 'US' && localeLanguageCode == \"en\" && ((isFirstStartup && !('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)) || 'bug-1675623-message-multi-stage-onboarding-set-default-experiment-release-82-85' in activeExperiments) && 'app.shield.optoutstudies.enabled'|preferenceValue",
```
Comment (punamdahiya) [inline messaging-experiments-82.json:1065]: realised on testing localeLanguageCode == \"en\" is failing targeting and should be updated to localeLanguageCode == 'en', will update  patch

**Comment by punamdahiya** [comment]
realised region == 'US' && localeLanguageCode == \"en\" , does not work for first run experiments as region is not available early enough, updating targeting to use locale == 'en-US'

**Pull request merged**